src/core/server/saved_objects/service/lib/filter_utils.ts

```diff
@@ -17,18 +17,18 @@
  * under the License.
  */
 
-import { fromKueryExpression, KueryNode, nodeTypes } from '@kbn/es-query';
 import { get, set } from 'lodash';
 import { SavedObjectsErrorHelpers } from './errors';
 import { IndexMapping } from '../../mappings';
+import { esKuery } from '../../../../../plugins/data/server';
 
 export const validateConvertFilterToKueryNode = (
   allowedTypes: string[],
   filter: string,
   indexMapping: IndexMapping
-): KueryNode => {
+): esKuery.KueryNode | undefined => {
```

Comment:
Why was this interface change introduced?

Reply:
I've set right  type for KueryNode. Initially it was:
![image](https://user-images.githubusercontent.com/20072247/69372671-32d94300-0cb3-11ea-8246-8ae3a513751d.png)